Sources/Plasma/CoreLib/plFileSystem.cpp

```diff
@@ -566,6 +569,24 @@ plFileName plFileSystem::GetCurrentAppPath()
         appPath = ST::string::from_wchar(path);
     }
 
+    return appPath;
+#elif HS_BUILD_FOR_MACOS
+    //Not sure how we'd ever patch the executable on iOS,
+    //this is really Mac specific
+    //This will only work if this is an app bundle
```

Comment:
This comment doesn't seem accurate anymore?

Reply:
Yep, removed!